### DISCUSSION TITLE
115怎样挂载啊

### DISCUSSION THREAD

**Discussion by dalaoJAck** [AlistGo/alist#2234] [Q&A]
115怎样挂载啊，我在alist文档并没有找到如何挂载它

**Comment by kevzcr** [comment]
你要等大神来回答，估计有得等了。。
1.浏览器登录115网盘。
2.按F12 打开 `devTools`  找到  `网络` 或` network `  选项。
3.按`ctrl+R` 多刷新几次后才能看到。
4.注意左边名称不一定，点开看，不管哪项，有`cookie`就行。 
5.打开alist 填入即可。 （115只要填写个cookie)

**Reply by dalaoJAck** [reply]
谢谢老哥

**Reply by simon333888** [reply]
115网盘登陆后隔段时间就需要重新登陆，有没有好的解决办法？

**Reply by simon333888** [reply]
115网盘登陆后隔段时间就需要重新登陆，有没有好的解决办法？

**Reply by SheltonZhu** [reply]
等扫码登录的版本，配合文档扫码登录，token的时间比较久

**Reply by i990049** [reply]
> 115网盘登陆后隔段时间就需要重新登陆，有没有好的解决办法？

学隔壁阿里云webdav设置成每隔几十分钟自动刷一次token就行了

**Reply by lyl980918** [reply]
请问2个问题，1.token是不是就是扫码获取的那个 填在令牌那里
2.如何设置成每几十分钟自动刷新一次token

**Comment by zqs12345677890** [comment]
> 115怎样挂载啊，我在alist文档并没有找到如何挂载它

我挂载成功了，但是打开没文件.........................

**Reply by lyl980918** [reply]
挂载后了可以用ip+端口号(5244)默认， 浏览器访问看看能不能访问成功，可以访问的话就可以通过支持Webdav的播放器播放 我用的是inufse

**Reply by 11ren** [reply]
最好打开无痕模式，取UID 、CID、 SEID 这三个数值，末尾不要符号
